### PR TITLE
Release/0.1.2

### PR DESCRIPTION
**Pull request by buehlere**
release 0.1.2

- fixed schema #159 
  - adjusted documentation gitaction for new schema
- added license #137 
- adding Code Owners #160 
- adding modules from develop

### CODE REVIEW

File: modules/msk/genomenexus/annotationpipeline/main.nf [hunk 1]
```diff
@@ -22,11 +22,11 @@ process GENOMENEXUS_ANNOTATIONPIPELINE {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     """
-    java -Xms${task.memory.toMega()/4}m -Xmx${task.memory.toGiga()}g -jar /genome-nexus-annotation-pipeline/annotationPipeline/target/annotationPipeline.jar --filename ${input_maf} --output-filename ${meta.id}_annotated.maf
```
Comment (nikhil) [inline modules/msk/genomenexus/annotationpipeline/main.nf:25]: This looks like a regression. I remember we added the java args for a bug fix. Also, it is moving the image version from 1.05 to 1.0.4

**Review by nikhil — CHANGES_REQUESTED** [review]

File: modules/msk/genomenexus/annotationpipeline/main.nf [hunk 2]
```diff
@@ -22,11 +22,11 @@ process GENOMENEXUS_ANNOTATIONPIPELINE {
...
+    java -jar /genome-nexus-annotation-pipeline/annotationPipeline/target/annotationPipeline.jar --filename ${input_maf} --output-filename ${meta.id}_annotated.maf
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        genomenexus: 'annotation pipeline version 1.0.5'
```
Comment (nikhil) [inline modules/msk/genomenexus/annotationpipeline/main.nf:29]: Not sure why the version is going down here

**Review by nikhil — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,2 +1,38 @@
 # Review from a member of the review-team is required
 * @mskcc-omics-workflows/reviewers
+./modules/msk/custom/splitfastqbylane/** @anoronh4 @mskcc-omics-workflows/reviewers
+./modules/msk/facets/** @nikhil @mskcc-omics-workflows/reviewers
+./modules/msk/gatk4/applybqsr/** @yocra3 @FriederikeHanssen @mskcc-omics-workflows/reviewers
```
Comment (rhshah) [inline .github/CODEOWNERS:5]: Just fyi we need to fix the meta yaml files here  by removing these users: @yocra3 @FriederikeHans

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
+./modules/msk/phylowgs/createinput/** @pintoa1-mskcc @mskcc-omics-workflows/reviewers
+./modules/msk/phylowgs/multievolve/** @nikhil @mskcc-omics-workflows/reviewers
+./modules/msk/phylowgs/parsecnvs/** @pintoa1-mskcc @mskcc-omics-workflows/reviewers
+./modules/msk/phylowgs/writeresults/** @nikhil @mskcc-omics-workflows/reviewers
+./modules/msk/polysolver/** @anoronh4 @mskcc-omics-workflows/reviewers
+./modules/msk/ppflagfixer/** @huyu335 @mskcc-omics-workflows/reviewers
+./modules/msk/pvmaf/concat/** @buehlere @mskcc-omics-workflows/reviewers
+./modules/msk/pvmaf/tagtraceback/** @buehlere @mskcc-omics-workflows/reviewers
+./modules/msk/rediscoverte/** @nikhil @mskcc-omics-workflows/reviewers
+./modules/msk/salmon/index/** @kevinmenden @drpatelh @nikhil @mskcc-omics-workflows/reviewers
```
Comment (rhshah) [inline .github/CODEOWNERS]: Again we need to fix the meta yaml files here  by removing these users: @kevinmenden @drpatelh

**Review by rhshah — APPROVED** [review]

**Review by nikhil — APPROVED** [review]

**Pull request merged**